sync/class.jetpack-sync-functions.php

```diff
@@ -99,17 +99,47 @@ public static function file_system_write_access() {
 	}
 
 	public static function home_url() {
-		return self::get_protocol_normalized_url(
-			'home_url',
-			self::normalize_www_in_url( 'home', 'home_url' )
-		);
+		if (
+			Jetpack_Constants::is_defined( 'JETPACK_SYNC_USE_RAW_URL' ) &&
```

Comment:
Any advantages if we flag this via Sync setting? so we can tweak it if needed via the sync settings endpoint?